bin/entrypoint

```diff
@@ -2,4 +2,8 @@
 
 set -e
 
-exec pensieve run "$@"
+if [ "$1" = "rerun_config_changed" ]; then
+    exec pensieve rerun_config_changed
+else
+    exec pensieve run "$@" --rerun-config-changed=true
```

Comment:
Do we want to re-run analyses for experiments with changed configs for the Airflow-scheduled pensieve? Probably I guess...

Reply:
That sounds right to me!